f8a_report/sentry_report_helper.py

```diff
@@ -0,0 +1,132 @@
+        """Init method for SentryReportHelper."""
+        self.s3 = S3Helper()
+        self.sentry_api_issues = os.getenv('SENTRY_API_ISSUES', 'https://errortracking'
+                                                                '.prod-preview.openshift'
+                                                                '.io/api/0/projects/openshift_io/'
+                                                                'fabric8-analytics-production/'
+                                                                'issues/')
+        self.sentry_api_tags = os.getenv('SENTRY_API_TAGS',
+                                         'https://errortracking.prod-preview'
+                                         '.openshift.io/api/0/issues/')
+        self.sentry_token = os.getenv('SENTRY_AUTH_TOKEN', '')
+
+    def retrieve_sentry_logs(self, start_date, end_date):
+        """Retrieve results for selected worker from RDB."""
+        result = {}
+        try:
+            # Invoke Sentry API to run the error collection
+            auth = 'Bearer {token}'.format(token=self.sentry_token)
+            resp = requests.get(url=self.sentry_api_issues + '?statsPeriod=24h',
+                                headers={"Authorization": auth})
+            # Check for status code
+            # If status is not success, log it as an error
+            if resp.status_code == 200:
+                logger.info('Successfully invoked Sentry API \n {resp}'.format(resp=resp.json()))
+                # associate the retrieved data to result
+                result = self.normalize_sentry_data(start_date, end_date, resp.json())
+            else:
+                logger.error('Error received from SSentry API \n {resp}'.format(resp=resp.json()))
+        except Exception as e:
+            logger.error('Unable to invoke Sentry API. Reason: %r' % e)
+        return result
+
+    def normalize_sentry_data(self, start_date, end_date, errorlogs):
+        """Retrieve results for selected worker from RDB."""
+        report_type = 'sentry-error-data'
+        report_name = dt.strptime(end_date, '%Y-%m-%d').strftime('%Y-%m-%d')
+        result = {
+            "error_report": {}
+        }
+        # Iterating all the error logs
+        for item in errorlogs:
+            errors = {}
+            events = self.retrieve_events(item['id'])
+            errors['id'] = item['id']
+            errors['lastSeen'] = item['lastSeen']
+            errors[events['pods_impacted']] = item['metadata']['type'] + ": " + \
+                item['metadata']['value'] if item['metadata'].get('type')\
+                else item['metadata']['title']
+            errors['stacktrace'] = events['stacktrace']
+            # Detecting the endpoint services
+            endpointpodname = events['pods_impacted'].split('-')
+            endpointpodname.pop(len(endpointpodname) - 1)
+            endpointpodname.pop(len(endpointpodname) - 1)
+            server_name = "-".join(endpointpodname)
```

Comment:
You can use more pythonic way then
`server_name = "-".join(endpointpodname.split("-")[:-2])`

Reply:
changed accordingly. thanks for suggestion